packages/barcode-scanning/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "pretty": true,
     "sourceMap": true,
     "strict": true,
-    "target": "es2017"
+    "target": "es2017",
+    "types": []
```

Comment:
Please revert that change. It's not related to this feature. You may need to install the latest version of `@types/node` as dev dependency if you have type issues.